test/unit/services/link_reporter_csv_service_test.rb

```diff
@@ -1,6 +1,9 @@
 require "test_helper"
 
 class LinkReporterCsvServiceTest < ActiveSupport::TestCase
+  include Rails.application.routes.url_helpers
```

Comment:
I wonder if this commit should be squashed into the previous one - keeping the implementation and test changes together?